src/LogWrapper.ts

```diff
@@ -27,36 +27,26 @@ function isMessageNoise(messageOrObject: MsgType[]) {
 interface HookshotLogInfo extends winston.Logform.TransformableInfo {
     data: MsgType[];
```

Comment:
`MsgType` is a union that includes `any`, which is what allowed this bug to slip through in the first place.

Is there a type that resolves to "any non-nullish values"?  Of course, the root of the problem is that `TransformableInfo.data` is `any`, so this might not help much...

Reply:
I mean really we're expecting to handle `any`, as any value should be loggable. I should probably just alias `MsgType` as `any`.